CryptoPrices/Sources/MyCoin/Sources/MyCoin/TimeFrameSectionView/TimeFrameSection.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  TimeFrameSection.swift
+//  MyCoin
+//
+//  Created by Khanh on 04/01/2023.
+//
+
+import Styleguide
+import SwiftUI
+
+struct TimeFrameSection: View {
+    
+    @State var selected: TimeFrameItem = .init(timeFrame: .oneDay)
```

Comment:
The `selected` should be marked as `private` since it's the internal `State` of the view 🙏